README.md

```diff
@@ -135,6 +157,33 @@ setupController: function(controller, model) {
   controller.set('products', model);
 }
 ```
+### event hooks
+
+ember-infinity provides a route mixin that enables us to hook into the following events:
+
+* **infinityModelUpdated**
+
+This gets fired up when data is loaded into the model.
```

Comment:
when you read this in Markdown - it looks like **infinityModelUpdated** is an argument for **infinityModelLoaded**

can we please remove the * from the function names so it is not rendered as an <li> tag?